src/common/components/hosts/SkipFormattingCheckbox.tsx

```diff
@@ -58,13 +75,11 @@ const SkipFormattingCheckbox: React.FC<SkipFormattingProps> = ({
     >
       <Checkbox
```

Comment:
Isn't the checkbox now about "formatting the disk" rather than "skipping formatting the disk"?

I think the names of the functions may be reversed.
So the component checkbox should be checked if the disk will be formatted